kogito-cloud-operator-build.jenkinsfile

```diff
@@ -30,7 +30,19 @@ pipeline {
                 }
             }
         }
-        
+        stage("Checkout pull request branch"){
+            when {
+                expression {
+                    return env.PULL_REQUEST != '';
+                }
+            }
+            steps {
+                dir("${WORKING_DIR}") {
+                    sh "git fetch origin pull/${PULL_REQUEST}/head:tempbranch"
+                    sh "git checkout tempbranch"
```

Comment:
maybe update the build name with the `PULL_REQUEST` name ?

Reply:
Yes, good point.

Reply:
+1 that would be nice